### PR TITLE
15 display a game board

### LINKED ISSUE #15: Display a game board
Display a game board that tracks players' locations. The locations should update when a player moves.

### PR DESCRIPTION
**Pull request by jsquires0**
Contains basic board functionality sufficient for demo -- tokens update on game board as moves are chosen. 
Verified  all rooms/halls are accessible and render tokens.
two_token_board

Limitations:

	-multiple tokens in same room overlap, obscuring each other (can fix or just avoid this situation in demo recording)
	-token placement affected by window size
	-starting locations not rendered. I'd vote not to bother unless there's a good reason.

Recommend merging and dealing with these separately after remaining core functionality is finished

### CODE REVIEW

File: web/components/Canvas/Board/Cards/Cards.module.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  // .card {
+  //   width: 100px;
+  //   position: relative;
+  //   transform: translateX(-50%);
+  // }
+
+  // > .card:first-child {
+  //   transform: translateX(0);
+  // }
+
```
Comment (benferro) [inline web/components/Canvas/Board/Cards/Cards.module.scss]: super-nit

```suggestion
```

File: web/components/Canvas/Sidebar/Chat/Chat.jsx [hunk 2]
```diff
@@ -30,13 +44,16 @@ function Chat() {
       });
     }
   }, [websocket]);
-
+  // TODO: action history isn't right..updates every time somebody chats
```
Comment (benferro) [inline web/components/Canvas/Sidebar/Chat/Chat.jsx:47]: Tbh I don't think we really need to preserve the action history. That was more of a feature for the demo imo

Comment (benferro) [inline web/components/Canvas/Sidebar/Chat/Chat.jsx:47]: Tbh I don't think we really need to preserve the action history. That was more of a feature for the demo imo

**Review by benferro — APPROVED** [review]
LGTM

File: web/components/Canvas/Sidebar/Chat/Chat.jsx (hunk 2 above)
Comment (jsquires0) [reply web/components/Canvas/Sidebar/Chat/Chat.jsx:47]: Its a requirement in the project description that "all players are notified of game state changes" so I think we need to keep it

**Pull request merged**